qiime_studio/api/plugins.py

```diff
@@ -51,7 +51,10 @@ def _build_data_dict(data):
             for name, type_ in value.signature.inputs.items()
         ]
         dict_[key]['parameters'] = [
-            {'name': name, 'type': repr(type_[0]), 'ast': type_[0].to_ast()}
+            {'name': name,
+             'type': repr(type_[0]),
+             'ast': type_[0].to_ast(),
+             'default': value.signature.defaults.get(name)}
```

Comment:
`.get()` will return `None`, so parameters without defaults will have `None` as a default. We currently don't support `None` as a default value but will support it soon (before workshop season). The CLI uses a sentinel to represent a truly missing default (see [here](https://github.com/qiime2/q2cli/blob/master/q2cli/handlers.py#L19)). That way parameters with default values of `None` can be represented as optional, and parameters without default values can be represented as required. Does this change make sense for the studio? @ebolyen interested in your input here if you have a moment.